### PR TITLE
Newtext+Images

### PR DESCRIPTION
**Pull request by jasminetey**
new images added were touched up and replaced placeholder text, changed the stairs photo to cropped one but you can change it back to the uncropped if it looks better 

the updated all text check failed since there were some problems with class not being className, should be ok now (on JoinAccordion.tsx)

### CODE REVIEW

**Comment by bergermaestro** [comment]
looks good! a few thoughts:
- When I made the Support page, the first paragraph was just a copy/paste from the about page, could you write something more relevant for that page, or just get rid of that first paragraph altogether?
image

---

(If you have time to do the rest of these on Friday that's great, otherwise don't worry about it and we can follow up in a separate pull request later)
- The new images look a bit dark, would it be possible to bump up the exposure on them?
- Please delete any of the picture files that you don't end up using, and the filenames of the ones you keep should be all lowercase
- What was your thought process on re-cropping the stairwell picture? I think it looks a bit strange with everyone so close to the bottom, but am open to it

File: src/components/JoinAccordion.tsx [hunk 1]
```diff
@@ -8,24 +8,73 @@ export const JoinFAQ = () => {
...
+
+      <AccordionItem>
+        <AccordionTrigger>Can I be a part of Formula Nano while on a work term?  </AccordionTrigger>
+        <AccordionContent>Of course! We accept students on both study and work terms. While our main lab facilities are located on campus at the University of Waterloo for hands-on research, team members also work remotely from all over the world. This flexibility allows you to join regardless of your location! </AccordionContent>
+      </AccordionItem>
+
+      <AccordionItem>
+        <AccordionTrigger> What can I do as a new team member?</AccordionTrigger>
+        <AccordionContent> As a new team member at Formula Nano, depending on our current project stage and your own preferences, you can contribute in numerous ways, including:  
+          <ul className= "list-disc list-inside ml-100 mt-2 ">
```
Comment (bergermaestro) [inline src/components/JoinAccordion.tsx]: `ml-100` is not a valid class - it's not doing anything

File: src/components/JoinAccordion.tsx [hunk 2]
```diff
@@ -8,24 +8,73 @@ export const JoinFAQ = () => {
...
+            <li>Business Development and Outreach: Support sponserships, finances, and promote our research and team via social media.</li>
+
+          </ul>
+            </AccordionContent>
+      </AccordionItem>
+
+      <AccordionItem>
+        <AccordionTrigger>What’s the process for joining Formula Nano? </AccordionTrigger>
+        <AccordionContent>
+          <ol className= "list-decimal list-inside ml-100 space-y-2 mt-2 ">
```
Comment (bergermaestro) [inline src/components/JoinAccordion.tsx]: same comment about `ml-100`

**Review by bergermaestro — CHANGES_REQUESTED** [review]

File: src/pages/index.astro [hunk 3]
```diff
@@ -82,27 +83,18 @@ import labImage from "../data/images/team_3.jpg";
...
-          />
-          <TimelineItem
-            date="April 2024"
-            title="Founded"
-            body="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed doeiusmod tempor incididunt ut labore et dolore magna aliqua."
-          />
+      
+        </h2>
+        <TimelineItem
+          date="August 2024"
```
Comment (bergermaestro) [inline src/pages/index.astro]: This should be Fall 2023

Comment (jasminetey) [reply src/pages/index.astro]: whoops supposed to write august 2023- ill change to fall 2023

**Review by bergermaestro — APPROVED** [review]

**Pull request merged**